lib/check-tags.js

```diff
@@ -0,0 +1,45 @@
+'use strict';
+const _ = require('lodash');
+
+function checkTags(options, mod, name, log) {
+  // Returns true if the module should be skipped.
+
+  if ((options.excludeTags.length && !options.includeTags.length && !mod.tags)
+    || (!options.includeTags.length && !options.excludeTags.length)) {
+    return false; // No checks to run.
+  } else if (options.includeTags.length && !options.excludeTags.length
```

Comment:
do we need to include `!options.excludeTags.length` here? If there are include tags, but no exclude tags, and there are no mod tags, then we definitely want to skip